ci/scripts/execute_redis_tests.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+cd ..
+git clone --config transfer.fsckObjects=false https://github.com/prettyClouds/redis.git
+cd redis
+git checkout tests-geode-redis
+
+JAVA_HOME="/usr/lib/jvm/java-11-openjdk-amd64"  \
```

Comment:
rather than hardcoding the JDK path, please use JAVA_HOME at this point, as you've already configured `ONLY_JDK: 11` and ci/scripts/execute_tests.sh sets up JAVA_HOME for the heavy lifter already

Reply:
Good call. I just updated it and exported the JAVA_HOME variable, definitely check it over and make sure it looks correct.

Reply:
Just pushed an update to execute_tests.sh that sets the JAVA_TEST_PATH variable so it's accessible inside the execute_redis_tests script.  It passes here: https://concourse.gemfire-ci.info/teams/developer/pipelines/sabbeypivotal-new-pr-pr/jobs/RedisTestOpenJDK11/builds/6

Reply:
If Robert were around today he might know a way to get it from gradle's testJVM property when gradle invokes the redis script, but this works too.